recipes/nyxus/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "nyxus" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sameeul/nyxus/archive/refs/heads/remove_submodules.zip
```

Comment:
one of the things from the deleted checklist in the issue template is about using official sources... you should try to do so if possible.

Reply:
Yes, we will move to official source as soon as one of my PR gets merged.

Reply:
Are your changes required for this recipe to build on conda-forge? Use a patch instead of publishing from an unofficial repository.

Reply:
My changes will get merged to the official repo today or tomorrow. As soon as that gets merged, I plan to update this. I resorted to this quick hack since I need to change both the build process in the actual repo and test it out with the recipe.